src/timing.c

```diff
@@ -188,6 +188,15 @@ JL_DLLEXPORT void jl_timing_show_method_instance(jl_method_instance_t *mi, jl_ti
                      jl_symbol_name(def->module->name));
 }
 
+JL_DLLEXPORT void jl_timing_show_method(jl_method_t *method, jl_timing_block_t *cur_block)
+{
+    jl_timing_show((jl_value_t *)method, cur_block);
```

Comment:
I recommend trying this out:
```c
JL_DLLEXPORT void jl_timing_show_quiet(jl_value_t *v, jl_timing_block_t *cur_block)
{
    jl_static_show_config_t config = { /* quiet */ 1 };
    return jl_timing_show_config(v, cur_block, config);
}
JL_DLLEXPORT void jl_timing_show(jl_value_t *v, jl_timing_block_t *cur_block)
{
    jl_static_show_config_t config = { /* quiet */ 0 };
    return jl_timing_show_config(v, cur_block, config);
}
static void _jl_timing_show_config(jl_value_t *v, jl_timing_block_t *cur_block, jl_static_show_config_t config)
{
#ifdef USE_TRACY
    ios_t buf;
    ios_mem(&buf, IOS_INLSIZE);
    buf.growable = 0; // Restrict to inline buffer to avoid allocation

    jl_static_show_x((JL_STREAM*)&buf, (jl_value_t *)method, config);
    if (buf.size == buf.maxsize)
        memset(&buf.buf[IOS_INLSIZE - 3], '.', 3);

    TracyCZoneText(*(cur_block->tracy_ctx), buf.buf, buf.size);
#endif
}
```

If you like the truncated output of `jl_timing_show_quiet`, we can use this in place of the needlessly-specific `jl_timing_show_func_sig`

Reply:
It was a bit messy with things being `static` inside `rtutils.c` etc. I'd suggest merging this a an incremental change and the printing of the signature can be tweaked independently.

Reply:
Sounds good to me